<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Admin_Umbrella_Add</class>
 <widget class="QWidget" name="Admin_Umbrella_Add">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>325</width>
    <height>417</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>雨伞添加或修改</string>
  </property>
  <widget class="QLineEdit" name="lineEdit_ubid">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>90</y>
     <width>171</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="placeholderText">
    <string>雨伞编号</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_ubtype">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>150</y>
     <width>171</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="placeholderText">
    <string>雨伞类型</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_placeid">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>210</y>
     <width>171</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="placeholderText">
    <string>地点编号</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_isb">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>270</y>
     <width>171</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="placeholderText">
    <string>是否借出</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_ok">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>330</y>
     <width>171</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>确定</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
